StyleGuide.sql

```diff
@@ -0,0 +1,73 @@
+-- # General
+
+-- Never mention database names in a query because they may not be the same across our platforms
+
+-- Capitalise keywords and function names
+-- Qualify table names with schema name
+-- Each field and part of statement should be on its own line
+-- Indent for readability
+UPDATE
+	[schema].[Table]
+SET
+	[FieldOne] = 'Thing',
+	[FieldTwo] = GETDATE()
+WHERE
+	[Name] = 'name'
+	AND [Description] = 'description'
+
+-- Id fields should be named [Id], never [TableNameId] or [TableName_Id]
+ALTER TABLE [schema].[Thing] ADD [Id] UNIQUEIDENTIFIER NOT NULL
+
+-- Queries should not be written in an idempotent way
+-- We run our queries in order through the UNiDAYS Updater
+-- We want queries to fail if the database is in such a state that the query can't be run
+
+
+-- # T-SQL
+
+-- Casing:
+-- Database name = UpperCamelCase
+-- Schema = lower case
+-- Table = UpperCamelCase
+-- Field = UpperCamelCase
+
+-- Names (field, table, schema) should be wrapped in []
+SELECT
+	[FieldOne],
+	[FieldTwo]

```

Comment:
comma should be at the beginning of next line, causes smaller diffs when new fields are added (1 line changed compared to 2)

Reply:
Done.